src/pyfaf/actions/releasedel.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (C) 2019  ABRT Team
+# Copyright (C) 2019  Red Hat, Inc.
+#
+# This file is part of faf.
+#
+# faf is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# faf is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with faf.  If not, see <http://www.gnu.org/licenses/>.
+
+from pyfaf.actions import Action
+from pyfaf.opsys import systems
+import pyfaf.storage as st
+from pyfaf.queries import (get_opsys_by_name,
+                           get_osrelease,
+                           get_empty_problems,
+                           get_builds_by_opsysrelease_id,
+                           delete_mantis_bugzilla,
+                           delete_bugzilla)
+
+
+class ReleaseDelete(Action):
+    name = "releasedel"
+
+    def run(self, cmdline, db):
+        if cmdline.opsys is None:
+            self.log_error("You must specify the operating system.")
+            return 1
+
+        if cmdline.opsys_release is None:
+            self.log_error("You must specify the operating system release.")
+            return 1
+
+        if cmdline.opsys not in systems:
+            self.log_error("Operating system '{0}' does not exist"
+                           .format(cmdline.opsys))
```

Comment:
Can you use lazy logging?
https://stackoverflow.com/questions/4148790/lazy-logger-message-string-evaluation